test/layer-download/config.yaml

```
@@ -0,0 +1,5 @@
+bobMinimumVersion: "0.16"
```

Not that it makes a difference but new tests should use the current version:

```suggestion
bobMinimumVersion: "0.18"
```

---

Changed.